app/assets/javascripts/components/samples.jsx

```diff
@@ -458,65 +522,163 @@ class Samples extends React.Component {
+                                {(column_name === name) ?
+                                  <i className="fa fa-check right"/> : null}
+                              </li>
+                              : <li key={`all-${i}`} className='selectable column_name' onClick={() => this.switchColumn(name, pos)}>
+                                { this.COLUMN_DISPLAY_MAP[name].display_name }
+                              </li>
+                          )
+                        })
+                        }
+                      </ul>
+                    </li>
+                  )
+                })
+                }
+              </ul>
             </div>
           </div>
-          <div className="col s2">Total reads</div>
-          <div className="col s2">Non-host reads</div>
-          <div className="col s1 center" rel='tooltip' data-placement='bottom' title='Passed quality control'>Passed QC</div>
-          <div className="col s1 center" rel='tooltip' data-placement='bottom' title='Duplicate compression ratio'>DCR</div>
-          <div className="col s2 status-dropdown" data-activates="dropdownstatus"><i className="status-filter fa fa-caret-down"></i>Status</div>
         </div>
       </div>
-    )
-    let project_id = this.state.selectedProjectId ? this.state.selectedProjectId : 'all'
-    let download_button = (
-      <a href={`/projects/${project_id}/csv`} className="download-project">
-        <div className="fa fa-cloud-download"/>
-        <div>Download table</div>
-      </a>
+    );
+    
+    const status_filter_dropdown = (
+      <ul id='dropdownstatus' className='status dropdown-content'>
+        <li><a className="title"><b>Filter by status</b></a></li>
+        <li className="divider"></li>
+        <li className="filter-item" data-status="WAITING" onClick={ this.handleStatusFilterSelect } ><a data-status="WAITING" className="filter-item waiting">Waiting</a><i data-status="WAITING" className="filter fa fa-check"></i></li>
+        <li className="filter-item" data-status="UPLOADING" onClick={ this.handleStatusFilterSelect }><a data-status="UPLOADING" className="filter-item uploading">In Progress</a><i data-status="UPLOADING"  className="filter fa fa-check"></i></li>
+        <li className="filter-item" data-status="CHECKED" onClick={ this.handleStatusFilterSelect }><a data-status="CHECKED" className="filter-item complete">Complete</a><i data-status="CHECKED" className="filter fa fa-check"></i></li>
+        <li className="filter-item" onClick={ this.handleStatusFilterSelect } data-status="FAILED" ><a data-status="FAILED" className="filter-item failed">Failed</a><i data-status="FAILED" className="filter fa fa-check"></i></li>
+        <li className="divider"></li>
+        <li className="filter-item" data-status="ALL" onClick={ this.handleStatusFilterSelect }><a data-status="ALL" className="filter-item all">All</a><i data-status="ALL" className="filter all fa fa-check"></i></li>
+      </ul>
```

Comment:
Looks like status_filter_dropdown isn't used anymore? (lines 658-662 are redundant with 616-621)
Please remove it here and in line 670